docs/source/examples/advanced_projects.rst

```diff
@@ -1,4 +1,90 @@
 Advanced Projects
 =================================
 
-This page is currently under construction. It will feature a selection of advanced projects built with manim.
+.. manim:: ExampleSineCurve
+
+    class ExampleSineCurve(Scene):
+        def construct(self):
+            self.show_axis()
+            self.show_circle()
+            self.move_dot_and_draw_curve()
+            self.wait()
+
+        def show_axis(self):
+            x_start = np.array([-6,0,0])
+            x_end = np.array([6,0,0])
+
+            y_start = np.array([-4,-2,0])
+            y_end = np.array([-4,2,0])
+
+            x_axis = Line(x_start, x_end)
+            y_axis = Line(y_start, y_end)
+
+            self.add(x_axis, y_axis)
+            self.add_x_labels()
```

Comment:
I got this one from the post here: https://www.reddit.com/r/manim/comments/iwcdfp/enjoy_my_collection_of_manim_code/ 
and just copy-pasted it.
Yep, this could also be done with GraphScene, but is it necessary?

Reply:
1. Did you ask the original author if we can use their code?
2. If you did, and the author said yes, we should provide due credit, and add a note or comment somewhere saying "this example provided by ____".
3. I think the purpose of the examples is to illustrate the intended use of manim. So if there is a GraphScene that could simplify this code, then we should use that, in order to illustrate the intended use of manim.

Reply:
1. Yes, I asked him and he agreed.
2. Good point! Let's discuss how we want to give credits to people on discord.
We could give him credits right here, after this example, or e.g. in one further document that is linked to the examples.
3. Also true. Maybe I will have a look on that.

Reply:
Cool cool cool, I'd say let's just add a comment in the example code itself saying something like "# taken from URL_HERE" or "# contributed by SOME_NAME".

Reply:
ok, done.